workspaces/vscode-cpp/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM prairielearn/workspace-vscode-base
+
+USER root
+RUN apt-get update && apt-get upgrade -y \
+    && apt-get install -y --no-install-recommends \
+    build-essential \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* \
+    && find /tmp -not -path /tmp -delete
+
+USER coder
+
+# After installing C/C++ we install some VS Code extensions.
+RUN code-server --disable-telemetry --force \
+    # a collection of useful C/C++ extensions
+    --install-extension franneck94.vscode-c-cpp-dev-extension-pack \
```

Comment:
Pending review by @jonatanschroeder to determine if these are indeed the extensions we want.